### PR TITLE
Update Stripe

### LINKED ISSUE #11936: Update: Stripe
### Brand Name

Stripe

### Official Resources for Icon and Color

logo kit: https://brandfolder.com/s/h3jbwkqfhbv9s9k4b9qpt3x

Main color is now purple: #7069fe
https://brandfolder.com/s/h3jbwkqfhbv9s9k4b9qpt3x

### Additional Comments

The color before was a blue: #008CDD.

### PR DESCRIPTION
**Pull request by koehlertimo**
**Issue:** closes #11936

**Popularity metric:** N/A

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
Changed color of stripe. Now it matches the current color theme

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -15138,7 +15138,7 @@
         },
         {
             "title": "Stripe",
-            "hex": "008CDD",
+            "hex": "7069FE",
```
Comment (uncenter) [inline _data/simple-icons.json:15141]:
```suggestion
            "hex": "635BFF",
```

`#635BFF` is the hex code I'm seeing used in the logo assets. Where did you get `#7069FE` from?

File: _data/simple-icons.json [hunk 2]
```diff
@@ -18092,4 +18092,4 @@
             "source": "https://www.zyte.com"
         }
     ]
-}
+}
```
Comment (uncenter) [inline _data/simple-icons.json:18095]: Please add the trailing newline back.

**Review by uncenter — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json [hunk 3]
```diff
@@ -15138,7 +15138,7 @@
         },
         {
             "title": "Stripe",
-            "hex": "008CDD",
+            "hex": "7069FE",
             "source": "https://stripe.com/about/resources"
```
Comment (uncenter) [inline _data/simple-icons.json:15142]:
```suggestion
            "source": "https://stripe.com/newsroom/information"
```
This link now redirects here.

**Comment by uncenter** [comment]
Please see https://github.com/simple-icons/simple-icons/issues/11936#issuecomment-2403759546. If possible could you update the icon as well? You can use the same source as is linked right now, https://stripe.com/newsroom/information. See https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#adding-or-updating-an-icon for instructions on how to update the SVG/icon.

**Review by adamrusted — APPROVED** [review]
Thanks for your work on this, @koehlertimo!

**Pull request merged**